docs/serving/load-balancing/README.md

```diff
@@ -0,0 +1,30 @@
+You can configure load balancing on Knative to place the activator in the request path to act as a load balancer.
+
+This guide explains how you can configure load balancing for your Knative system using the activator.
+
+## Prerequisites
+- Ensure that load balancing using an ingress gateway (for example, Istio or Kourier) is not enabled.
+- Ensure that individual pod addressability is enabled.
+
+## Activator pod selection
+In general, the system will perform best if the number of existing pods is larger than the number of activators, and those numbers divide equally.
+<!--TODO: Add better documentation about what the activator is; explain the components of load balancing; maybe add a diagram-->
+Knative assigns a subset of activators for each revision, depending on the revision size. More revision pods will mean a greater number of activators for that revision. Activators are scaled horizontally, so there may be multiple activators in a deployment.
+
+The activator load balancing algorithm works as follows:
+- If concurrency is unlimited, the request is sent to a random pod.
+- If concurrency is set to a limited value, the activator will send the request to the first pod that has capacity.
```

Comment:
This is now upgraded to
`the request to the first pod that has capacity if container concurrency is very small, otherwise requests will be balanced in a round robin fashion`.

Reply:
Define very small though? What's the actual cut off concurrency wise before it does round robin?

Reply:
Added a note since I think we should address this better but I don't want to block the PR any longer on it